adafruit_cap1188/cap1188.py

```diff
@@ -63,16 +63,21 @@
                               const(0x15),
                               const(0x16),
                               const(0x17))
+CAP1188_SENSITIVTY          = const(0x1F)
```

Comment:
Shouldn't these all start with `_` to save memory? I realize you didn't change it here but its probably worth doing. Variable names take a lot of memory.

Reply:
Yep. Just not being good at being consistent. Names changed.